compaction/time_window_compaction_strategy.cc

```diff
@@ -220,14 +220,11 @@ time_window_compaction_strategy::get_sstables_for_compaction(column_family& cf,
     }
 
     if (!expired.empty()) {
-        auto is_expired = [&] (const shared_sstable& s) { return expired.contains(s); };
-        candidates.erase(boost::remove_if(candidates, is_expired), candidates.end());
+        clogger.debug("Going to compact {} expired sstables", expired.size());
+        return compaction_descriptor(has_only_fully_expired::yes, std::vector<shared_sstable>(expired.begin(), expired.end()), cf.get_sstable_set(), service::get_local_compaction_priority());
```

Comment:
@raphaelsc do we check somewhere that the fully expired tombstones are older than all other data? Otherwise we can have data resurrection.